tasks/README.md

```diff
@@ -296,3 +296,6 @@ Name | Summary | Category
 `task380_boolq_yes_no_question` | Given a passage and a yes/no question, answer the question based on the passage | Answer Generation
 `task381_boolq_question_generation` | Given a passage, generate a yes/no question that can be answered based on the passage | Question Generation
 `task382_hybridqa_answer_generation` | Given a question about part-of-speech tag of a word in the question, answer the question | Answer Generation
+`task391_causal_relationship` | Given two sentences, decide whether there is a cause and effect relationship between them, and the second one is a result of the first one. | Answer Generation
```

Comment:
this could be written as
`Given two sentences, decide whether the second sentence can be the result of the first one`
also please remove the "."